waspc/src/Wasp/Generator/DbGenerator/Jobs.hs

```diff
@@ -85,43 +85,54 @@ asPrismaCliArgs migrateArgs = do
 -- Because of the --exit-code flag, it changes the exit code behavior
 -- to signal if the diff is empty or not (Empty: 0, Error: 1, Not empty: 2)
 migrateDiff :: Path' Abs (Dir ProjectRootDir) -> J.Job
-migrateDiff projectDir = runPrismaCommandAsDbJob projectDir $ \schema ->
-  [ "migrate",
-    "diff",
-    "--from-schema-datamodel",
-    SP.fromAbsFile schema,
-    "--to-schema-datasource",
-    SP.fromAbsFile schema,
-    "--exit-code"
-  ]
+migrateDiff projectRootDir =
+  runPrismaCommandAsJob
+    projectRootDir
+    [ "migrate",
+      "diff",
+      "--from-schema-datamodel",
+      SP.fromAbsFile schema,
+      "--to-schema-datasource",
+      SP.fromAbsFile schema,
+      "--exit-code"
+    ]
+  where
+    schema = projectRootDir </> dbSchemaFileInProjectRootDir
 
 -- | Checks to see if all migrations are applied to the DB.
 -- An exit code of 0 means we successfully verified all migrations are applied.
 -- An exit code of 1 could mean either: (a) there was a DB connection error,
 -- or (b) there are pending migrations to apply.
 -- Therefore, this should be checked **after** a command that ensures connectivity.
 migrateStatus :: Path' Abs (Dir ProjectRootDir) -> J.Job
-migrateStatus projectDir = runPrismaCommandAsDbJob projectDir $ \schema ->
-  ["migrate", "status", "--schema", SP.fromAbsFile schema]
+migrateStatus projectRootDir =
+  runPrismaCommandAsJob
+    projectRootDir
+    ["migrate", "status", "--schema", SP.fromAbsFile schema]
+  where
+    schema = projectRootDir </> dbSchemaFileInProjectRootDir
```

Comment:
This is releating itself quite a bit, what about having a function `getSchemaPath :: Path' Abs (Dir ProjectRootDir) -> ...`, would that be a bit nicer? Or not much of an improvement?

Reply:
Sure, I'll add it in.